### ISSUE TITLE
Publish library on NPM

### ISSUE THREAD

**Issue by benjamindulau** [lifelynl/focuspoint#1]
Hey guys,

Could you publish this library on NPM please?
That would ease usage with CommonJS and Browserify!

Thanks

**Comment by jessedvrs** [comment]
Done :)
https://www.npmjs.com/package/focuspoint
`npm install focuspoint`

**Comment by benjamindulau** [comment]
Sweet ;-)
But when I try ton import the module I get the following error:

```
Error: Cannot find module 'load-grunt-config' from '/vagrant/node_modules/focuspoint'
```

Code:

``` javascript
import Focuspoint from 'focuspoint';
```

**Comment by jessedvrs** [comment]
Looks like this was a problem with my package.json. I fixed the problem in commit bfcc8266b958aa21aeb21dffbea5579d6448a60f and re-published the npm package under 1.0.6. Please run `npm update focuspoint`.

**Comment by benjamindulau** [comment]
Nice! works perfectly. Thanks ;-)
